template_config.yml

```diff
@@ -56,6 +56,7 @@ pulp_settings:
   allowed_import_paths: /tmp
   ansible_api_hostname: https://pulp:443
   ansible_collect_download_log: true
+  ansible_collect_download_count: true
```

Comment:
It would be good to have these settings only on one of the test scenarios.
(Their tests would need to be optionally skipped though.)

Reply:
I'm not exactly sure how to accomplish this, is there an existing example I could use for inspiration?  Assuming this is regarding both `ansible_collect_download_log` and `ansible_collection_download_count`?

Reply:
You would move these two lines to the specific scenario settings below (`pulp_settings_azure`, ...). I think we can have all possible 4 combinations. ;)
For the tests, there is a `pulp_settings` fixture and you can conditionally `pytest.skip("")` a test.